src/main/java/com/hydratereminder/HydrateReminderTimer.java

```diff
@@ -46,6 +46,8 @@
 @ToString
 public class HydrateReminderTimer extends InfoBox
 {
+    private static final String TOOLTIP_INFORMATION = "Time until next hydration break.";
```

Comment:
Just a minor nitpick. Can you remove the period at the end?

Otherwise, all your changes look good to me!